wp-pgp-encrypted-emails.php

```diff
@@ -340,34 +458,90 @@ public static function registerAdminSettings () {
             self::$meta_key_sign_for_unknown_recipients,
             __('Sign email sent to unrecognized addresses', 'wp-pgp-encrypted-emails'),
             array(__CLASS__, 'renderSignForUnknownRecipients'),
-            'general',
-            'default',
+            'wp-pgp-encrypted-emails',
+            'wp-pgp-encrypted-emails',
             array(
                 'label_for' => self::$meta_key_sign_for_unknown_recipients
             )
         );
         register_setting(
-            'general',
+            'wp-pgp-encrypted-emails',
             self::$meta_key_sign_for_unknown_recipients,
             array(__CLASS__, 'sanitizeCheckBox')
         );
 
+        // S/MIME signing keypair
+        /* Not implemented at this stage
```

Comment:
Code that isn't implemented shouldn't be commented out, it just shouldn't be included in the first place. :) Would you be willing to remove this portion until it's ready?